src/main/resources/init_db.sql

```diff
@@ -0,0 +1,8 @@
+CREATE DATABASE IF NOT EXISTS book;
+USE book;
+CREATE TABLE IF NOT EXISTS books
+(
+    id BIGINT NOT NULL AUTO_INCREMENT PRIMARY KEY,
```

Comment:
FYI: PRIMARY KEY ensures that the ID is non-null and unique, so there is no need to mention NOT NULL.